src/currentprocess/filesource.rs

```diff
@@ -79,71 +79,38 @@ impl StdinSource for super::TestProcess {
     }
 }
 
-// -------------- stdout -------------------------------
-
-pub trait Isatty {
-    fn isatty(&self) -> bool;
-}
-
-/// Stand-in for std::io::StdoutLock
+/// Allows you to write to a terminal or other output stream.
```

Comment:
I think this loses key information - std::io::StdoutLock is a crucial part of the behaviour this is emulating. I'm happy for more information to be added to the docstring, but I don't think replacing what is there with what you've got is sufficient.

Reply:
Added more info.